R/add_sda_target.R

```diff
@@ -0,0 +1,185 @@
+#' Add portfolio level SDA target
```

Comment:
Haven't read the documentation yet. Maybe Vincent would be a good person to review it.